document/CoreFormat.mdk

```diff
@@ -1891,6 +1989,23 @@ a stable, opaque identifier for the rule.
 `"CA2101"`
 ~
 
+~ IsoNote
+Rule identifiers must be stable for two reasons:
+
+1. So specific checks can be disabled in build automation scripts, without the risks
+of the scripts breaking if a rule id changes.
+2. So result management systems can compare results from one run to the next,
+without erroneously designating results as "new" because a rule id has changed.
+
+Rule identifiers should be opaque -- that is, they should not convey information to
+a user -- because a rule's implementation might change over time.
```

Comment:
As we did that recently with the FxCop XML rules. But I'm not sure how that fits into a note that explains why rule ids should be stable and opaque. In the scenario you're describing, the rule ids were _not_ in fact stable.